dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -47,6 +89,29 @@ def refute_estimate(self, *args, **kwargs):
                                     n_samples=self._sample_size,
                                     random_state=self._random_state )
 
+            if self._chosen_variables is not None:

```

Comment:
what happens if it is None? I presume that we choose all variables and add noise to it?

Reply:
There are two ways to go about it
Option 1:
Add noise to all variables
Option 2:
Add noise to no variable and run the existing functionality of bootstrapping the dataset.

At present, I have implemented Option 2.

Reply:
Okay, but there is quite an inconvenience in this. Suppose three common cases: 
1. user wants to add noise to no variable
2. user wants to add noise to all variables
3. user wants to add noise to specific variables

Specifying a list for the 3rd one is okay. But I feel that there should be easy defaults for 1 and 2. Currently, would a user need to copy the entire list of variables to achieve 2?
One solution is to have string values like "none" or "all", but I don't like that solution a lot since it creates arbitrary string values.
Any ideas?

Reply:
Another option is to add a boolean parameter that allows us to add noise to all variables.